### PR TITLE
[13.0][MIG] purchase_order_product_recommendation_forecast: MIgration to 13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31769

please @chienandalu @carlosdauden review this

### CODE REVIEW

**Comment by pedrobaeza** [comment]
/ocabot migration purchase_order_product_recommendation_forecast

**Review by carlosdauden — APPROVED** [review]

File: purchase_order_product_recommendation/wizards/purchase_order_recommendation.py [hunk 1]
```diff
@@ -108,10 +108,14 @@ def _get_products(self):
     def _get_move_line_domain(self, products, src, dst):
         """Allows to easily extend the domain by third modules"""
         combine = datetime.combine
+        # We can receive a context to be able to get different dates with
+        # the same wizard attributes, for example comparing periods
+        date_begin = self.env.context.get("period_date_begin", self.date_begin)
+        date_end = self.env.context.get("period_date_end", self.date_end)
```
Comment (chienandalu) [inline purchase_order_product_recommendation/wizards/purchase_order_recommendation.py:114]: Put this in a different commit better

File: purchase_order_product_recommendation_forecast/readme/USAGE.rst [hunk 2]
```diff
@@ -1,6 +1,6 @@
 To use this module, you need to:
 
-#. Create a new purchase order.
+#. Create a new uprchase order.
```
Comment (chienandalu) [inline purchase_order_product_recommendation_forecast/readme/USAGE.rst]: Do you think it better like this? :smile:

**Comment by CarlosRoca13** [comment]
Changed @chienandalu

**Review by chienandalu — APPROVED** [review]
